packages/flutter_tools/lib/src/test/runner.dart

```diff
@@ -88,6 +89,7 @@ class _FlutterTestRunnerImpl implements FlutterTestRunner {
     String randomSeed,
     @required List<String> extraFrontEndOptions,
     bool nullAssertions = false,
+    BuildInfo buildInfo // TODO(jonahwilliams): make the default argument
```

Comment:
nit: missing comma

Reply:
I'm about to head out, if you want to apply this change and re-run CI that is fine - otherwise I can clean this up next week when I make these required and update g3

Reply:
later is fine.